src/applications/hca/utils/validation.js

```diff
@@ -13,18 +13,9 @@ export function validateServiceDates(
 ) {
   const fromDate = convertToDateField(lastEntryDate);
   const toDate = convertToDateField(lastDischargeDate);
-  const yearFromToday = endOfDay(add(new Date(), { years: 1 }));
+  const yearFromToday = add(new Date(), { years: 1 });
   const endDate = format(yearFromToday, 'MMMM d, yyyy');
 
-  if (
```

Comment:
Just changing the order of operations to check the entry date against the Veteran birthdate first, since that is the order it would likely be input.

Reply:
Would it make sense to add a test for this? If we switch it back nothings fails at the moment.